facade/src/main/java/org/jboss/pnc/facade/providers/SCMRepositoryProviderImpl.java

```diff
@@ -186,13 +186,21 @@ public RepositoryCreationResponse createSCMRepository(
     @Override
     public SCMRepository update(String id, SCMRepository restEntity) {
```

Comment:
You can just delete this method now :D No need to override. The update from AbstractProvider is the same after all now.